### ISSUE TITLE
17 news

### ISSUE THREAD

**Issue by DaMacFunkin** [koying/SPMC#893]
Any news of a v17 beta yet?
Thanks.

**Comment by slipk3** [comment]
+1 for SPMC v17
I love SPMC 16.6-81, so smooth and great on my Shield TV
But at the same time more and more things no longer support v16.
I have both of SPMC 16.6-81 and Kodi 17.1 installed and switch between them as needed.

I would love to try out the SPMC v17 beta if it exists and stick with it. I am ok with some bugs on it as I trust Koying will improve it over time. Thanks.

**Comment by CLFNole** [comment]
Where did you get 16.6-81? What's different about it from 16.6.0?

**Comment by ghost** [comment]
16.6-81 still no 'official' release yet.
No new official SPMC releases for 2 months now.
I and many others await news on 17 or anything SPMC related.
@koying hope to hear something soon.

**Comment by rcrh** [comment]
I know there's a development time line and any 17.x version is officially awhile off.  But, I'd like to throw in another voice for a move of the database versions to make spmc 16.x compatible with kodi 17.x

**Comment by slipk3** [comment]
v16.6-81 is here: http://download.semperpax.com/spmc/
look for the 03/31/2017 release
also, see https://github.com/koying/SPMC/issues/871

**Comment by CLFNole** [comment]
Thanks slipk3...do you like it over 16.0.0?

**Comment by HeXDeMoN** [comment]
Koying updated the python version with that version so most add-ons that are broke are now fixed . Definitely​ worth using over 16.6.0

**Comment by CLFNole** [comment]
Cool...thanks guys.

**Comment by CLFNole** [comment]
It's the arm version for Shield correct?

**Comment by CLFNole** [comment]
Actually I think it's the aarch64 version for Shield (recently got it) as I think i used and arm version previously for my Sony Android tv.

**Comment by DaMacFunkin** [comment]
It's actually v17 database compatibility i would like so i can update my Zidoo x9s to ZDMC 17.1 and have all my devices use the same database.
Regards.

**Comment by janolapin2016** [comment]
Did you try FTMC ? FTMC include python 2.7 and many other improvment that SPMC doesnt have ... FTMC is freaktab fork. google it.

**Comment by oOflyeyesOo** [comment]
What are some differences with FTMC, couldn't find too much.

**Comment by janolapin2016** [comment]
Python 2.7 first 
See thread here http://freaktab.com/forum/tv-player-support/general-tv-player-dicussions/xbmc-talk/575588-ftmc

**Comment by oOflyeyesOo** [comment]
That is one thing I remembered, but I guess now it goes to what are the benefits of 2.7 python.

**Comment by janolapin2016** [comment]
FTMC is specificaly designed for AMLogic and Rockchip devices and it run as hell

**Comment by cowballz** [comment]
FTMC doesn't have DASH support for you tube. Dead fish in water for me when I tested it

**Comment by janolapin2016** [comment]
I have FTMC installed on two device and they both play YouTube as hell so there is something wrong with your configuration or device

**Comment by cowballz** [comment]
YouTube works in 720p only on that build due to no Dash support

**Comment by janolapin2016** [comment]
are you sure ? I mean there is no difference for me

**Comment by cowballz** [comment]
Yes hit the menu button and it will show you info about the video. 1280x720

**Comment by ghost** [comment]
big plus for FTMC is this option:
`[Extra] Disabled External Player confirmation dialog. Like this theirs no more need to press 'OK' every time it starts/stops the play of a video file! ( Thks mo123 for the tip ;) )`

For me very important (using a Q10 Pro and external player)

**Comment by DaMacFunkin** [comment]
Does FTMC specifically support AFTV2 chipset?

**Comment by ghost** [comment]
They have 2 versions. Standard en RK. I really like to use SPMC, but this is annoying  if you use a external player

**Comment by ghost** [comment]
They have 2 versions. Standard en RK. I really like to use SPMC, but this is annoying  if you use a external player

**Comment by DaMacFunkin** [comment]
What i am saying is does FTMC have refresh rate switching and Hardware acceleration for AFTV2?

**Comment by ghost** [comment]
@koying 
Have we any news on new version yet ?

**Comment by cowballz** [comment]
Has the SPMC project been abandoned?

**Comment by oOflyeyesOo** [comment]
It hasn't. Koying made a few comments a couple days ago. I know that he was at the Kodi devcon. I'm sure that took up some time. Possibly new ideas.

**Comment by ghost** [comment]
> I know that he was at the Kodi devcon <

Didn't @koying  actually leave the Kodi team not so long ago ?
This was actually stated that he left.
So why would he take time to be there ?
Must say this is more than confusing, if he's abandoned SPMC and gone back to Kodi ?
Who knows ???

He certainly doesn't seem to respond HERE much these days.

**Comment by Nerto** [comment]
> I know that he was at the Kodi devcon <
Didn't @koying actually leave the Kodi team not so long ago ?
This was actually stated that he left.
So why would he take time to be there ?
Must say this is more than confusing, if he's abandoned SPMC and gone back to Kodi ?
Who knows ???

He certainly doesn't seem to respond HERE much these days.

He is working on kodi 17. You can see it at https://github.com/koying/SPMC/branches at spmc-krypton last update 2 days.

**Comment by ghost** [comment]
@Nerto 
Thanks for info, so very pleased to be hearing this good news.

**Comment by sandog78** [comment]
I'm reading that we should be using 17.3 now for certain streams is spmc  have that capability of playing them

**Comment by sammarbella** [comment]
We must remember Koying develops SPMC spending his personal time on this project for FREE.
I prefer he spends his time developing SPMC than replying to us.
As Nerto said he is working on Krypton branch and he also had time to make a new Android Jarvis build 2 days ago: 16.7.0.

**Comment by rcrh** [comment]
Where did you find a 16.7.0 release?  The "releases" page (https://github.com/koying/SPMC/releases) only has 16.6.0

**Comment by flipside101** [comment]
usual place , the dl site http://download.semperpax.com/spmc/

**Comment by DaMacFunkin** [comment]
Mmmm i wonder what the release notes are?

**Comment by rcrh** [comment]
So the downloads links on github are being maintained?

**Comment by ghost** [comment]
Okie Dokie then another 16 Jarvis build is out BUT ...
Think we should keep this thread 17 relevant as this was @DaMacFunkin the OP's question :)
